job-service/src/main/java/com/hpe/caf/services/job/api/ApiExceptionMapper.java

```diff
@@ -41,13 +44,20 @@
     @Override
     public Response toResponse(Exception exception) {
         final Response.Status httpStatus;
-        if (exception instanceof BadRequestException) {
+        if (exception instanceof BadRequestException ||
+            exception instanceof UnrecognizedPropertyException
```

Comment:
Why are we adding this?

Reply:
If we pass an extra parameter for creating the job, we get an UnrecognizedPropertyException, which extends IOException. So, the way it's, it would be returning a 503. Adding that check allows to return a 400 which seems more appropriate.